modules/mauth-authenticator-apachehttp/README.md

```diff
@@ -49,14 +51,13 @@ This is an implementation of Medidata Authentication Client Authenticator to val
         @Override
```

Comment:
Instead of writing a sample code here, could we create an example project that user can see possibly another module that's not published